### PR TITLE
Assets and Elastic Search error in docker

### PR DESCRIPTION
**Pull request by Jeshan7**
**Motivation and Context**
 
 Docker setup of Aviacommerce on Linux machine is facing the below two problems
   + Elastic Search was exiting with code 78. This was due to low 'max file descriptor size' and low 'max 
        virtual memory'.
   + Assets were not getting installed automatically. 

  **Describe your changes**
   + Made changes in Dockerfile-dev to resolve the assets problem.
   + Made changes in  docker-compose-dev.yml to resolve elastic search problem as it
    was getting stopped due to low descriptor size

## This change addresses the need by:

[delivers #162446059]

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

### CODE REVIEW

File: apps/snitch_core/config/dev.exs [hunk 1]
```diff
@@ -6,7 +6,7 @@ config :snitch_core, Snitch.Repo,
   username: "postgres",
   password: "postgres",
   database: "snitch_dev",
-  hostname: "localhost",
+  hostname: "db",
```
Comment (arjun289) [inline apps/snitch_core/config/dev.exs]: This should be picked up from system env otherwise will fail for manual setup

Comment (ashish173) [reply apps/snitch_core/config/dev.exs]: @Jeshan7 @vikramjadon921995 please look into it.

Comment (vikramjadon921995) [reply apps/snitch_core/config/dev.exs]: @ashish173 Yes. I and @Jeshan7 have discussed this and he is currently working on the changes.

**Review by arjun289 — COMMENTED** [review]
LGTM :+1:

**Review by vikramjadon921995 — APPROVED** [review]

**Pull request merged**